ilastik/workflows/objectClassification/objectClassificationWorkflow.py

```diff
@@ -670,38 +708,28 @@ class ObjectClassificationWorkflowBinary(ObjectClassificationWorkflow):
     workflowName = "Object Classification (from binary image)"
```

Comment:
the `ObjectClassificationWorkflowBinary` lacks an implementation of `createInputApplets`, I assume that's some leftover stuff from previous comments (updating it to `abstactmethod`)

Reply:
No idea how i missed that, but yeah. Fixed.